sycl/test-e2e/NonUniformGroups/chunk.cpp

```diff
@@ -13,19 +13,19 @@
 #include <vector>
 namespace syclex = sycl::ext::oneapi::experimental;
```

Comment:
This test has a missing requirement. As the spec says, the feature is only available if `ext_oneapi_chunk` is supported by a device.

Therefore, there should be a line `REQUIRES: aspect-ext_oneapi_chunk` and corresponding aspect should be updated